composer/core/types.py

```diff
@@ -77,6 +78,22 @@ def as_batch_pair(batch: Batch) -> BatchPair:
     return batch
 
 
+def _split_fn(batch: Batch, n_microbatches: int) -> List[Batch]:
```

Comment:
This function was moved to the `DataSpec` class; I presume it was left over from the merge?